mmv1/products/blockchainnodeengine/BlockchainNodes.yaml

```
@@ -0,0 +1,210 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BlockchainNodes'
+base_url: projects/{{project}}/locations/{{location}}/blockchainNodes
+self_link: 'projects/{{project}}/locations/{{location}}/blockchainNodes/{{blockchain_node_id}}'
+import_format: ['projects/{{project}}/locations/{{location}}/blockchainNodes/{{blockchain_node_id}}']
+create_url: projects/{{project}}/locations/{{location}}/blockchainNodes?blockchain_node_id={{blockchain_node_id}}
+description: |
+  A representation of a blockchain node.
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 45
+  delete_minutes: 35
+
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/blockchain-node-engine'
+  api: 'https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'blockchain_nodes_basic'
+    primary_resource_id: 'default_node'
+    vars:
+      blockchain_node_id: 'blockchain_basic_node'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'blockchain_nodes_geth_details'
+    primary_resource_id: 'default_node_geth'
+    vars:
+      blockchain_node_id: 'blockchain_geth_node'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: location
+    description: |
+      Location of Blockchain Node being created.
+    immutable: true
+    required: true
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: blockchainNodeId
+    url_param_only: true
+    required: true
+    description: |
+      ID of the requesting object.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The number of the google project that firebase is enabled on.
```

```suggestion
      The fully qualified name of the blockchain node. e.g. projects/my-project/locations/us-central1/blockchainNodes/my-node.
```

Looks like a copy-paste error - [here's the info from the docs](https://cloud.google.com/blockchain-node-engine/docs/reference/rest/v1/projects.locations.blockchainNodes#:~:text=Output%20only.%20The%20fully%20qualified%20name%20of%20the%20blockchain%20node.%20e.g.%20projects/my%2Dproject/locations/us%2Dcentral1/blockchainNodes/my%2Dnode.)